docs/site/Example1.md

```diff
@@ -0,0 +1,29 @@
+---
+lang: en
+title: Examples
+keywords: LoopBack 4.0
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Examples.html
```

Comment:
I am confused about the purpose of this file. It has filename `Example1.md`, uses the same permalink `/doc/en/lb4/Examples.html` as the file `Example.md` below, and described `rpc-server` example only.

Maybe this some sort of a back-up or earlier draft that should have been deleted before opening the pull request?

Anyhow, can we remove this file?

Reply:
Yes I did remove it and somehow it was added back in after my many conflict resolutions! It was an interim file.